examples/v1beta1/taskruns/workspace-in-sidecar.yaml

```diff
@@ -19,6 +19,10 @@ spec:
     - name: signals
     steps:
     - image: alpine:3.12.0
+      resources:
```

Comment:
@sbwsg I am wondering why these changes are showing up here 🤔  These were already a part of previous [PR](https://github.com/tektoncd/pipeline/commit/565c55b30e422fc7fca42bc122561aa231106204#diff-d3a6963f568541460e24acddf2bcc5d3337bbbc8632e91a63ee59c6084891a2eR22-R25) you created.

Reply:
and which was merged ...